src/components/learner-credit-management/data/hooks/useBudgetDetailActivityOverview.js

```diff
@@ -4,14 +4,22 @@ import useBudgetId from './useBudgetId';
 import useSubsidyAccessPolicy from './useSubsidyAccessPolicy';
 import { learnerCreditManagementQueryKeys } from '../constants';
 
-const useBudgetDetailActivityOverview = ({ enterpriseUUID, isTopDownAssignmentEnabled }) => {
+const useBudgetDetailActivityOverview = ({
+  enterpriseUUID, isTopDownAssignmentEnabled, paramBudgetId = '', paramSubsidyAccessPolicyId = '',
+}) => {
+  let budgetID = paramBudgetId;
+  let subsidyAccessPolicyID = paramSubsidyAccessPolicyId;
   const { budgetId, subsidyAccessPolicyId } = useBudgetId();
```

Comment:
nit: Having two variables `budgetID`, `budgetId` that differ only by casing is confusing